### PR TITLE
Updated styling

### PR DESCRIPTION
**Pull request by taahirisaacs**
@craig-mulligan @shaunmulligan

### CODE REVIEW

**Comment by craig-mulligan** [comment]
Made one or two comments, after that we are good to merge :)

File: static/css/main.css [hunk 1]
```diff
@@ -340,7 +348,3 @@ ul.expand {
   padding-top: 50px;
   margin-top: 0;
 }
-
-.shadow {
```
Comment (emirotin) [inline static/css/main.css]: better to cleanup this class from where it's used

**Comment by emirotin** [comment]
LGTM

**Comment by craig-mulligan** [comment]
Good to merge @taahirisaacs. :+1:

**Pull request merged**